### DISCUSSION TITLE
Image background returns black

### DISCUSSION THREAD

**Discussion by Pablituuu** [pymupdf/PyMuPDF#3068] [Q&A] [answered]
The background of a PDF is white and when I extract the images, it returns black.

I have created an environment where you can see the error, in the following link:
Around: https://codesandbox.io/p/devbox/issue-pdf-2p9q6s

The pdf that indicates this error is the following:
PDF: [Big_Foot.pdf](https://github.com/pymupdf/PyMuPDF/files/13994442/Big_Foot.pdf)

I tried to remove the mask that I saw according to other previously opened cases, but this one only returns a mask and I cannot make it transparent.

This is the original pdf.
![image](https://github.com/pymupdf/PyMuPDF/assets/61519207/cc450cdc-9ec4-49e0-9972-9c8894e1fe03)

And this is the one that returns to me.
![img](https://github.com/pymupdf/PyMuPDF/assets/61519207/dc853940-19c2-49be-8ec3-60b1360df021)

**Comment by JorjMcKie** [comment, marked as answer]
You did not tell us the code you tried. This code works and is also [documented!](https://pymupdf.readthedocs.io/en/latest/recipes-images.html#how-to-handle-image-masks)
```python
import fitz
doc=fitz.open("Big_Foot.pdf")
page=doc[0]
page.get_images()
[(12, 43, 2406, 2089, 8, 'DeviceRGB', '', 'X4', 'FlateDecode')]
basepix = fitz.Pixmap(doc,12)
maskpix = fitz.Pixmap(doc,43)
pix = fitz.Pixmap(basepix, maskpix)
pix.save("bigfoot.png")
```

**Reply by Pablituuu** [reply]
Thanks friend, the code is in the following codesandbox URL, but the solution worked for me, thanks again

https://codesandbox.io/p/devbox/issue-pdf-2p9q6s
